package.json

```diff
@@ -12,6 +12,8 @@
     "browserify": "^13.0.0",
     "chai": "^3.5.0",
     "coveralls": "^2.11.8",
+    "del": "^2.2.0",
+    "gh-pages": "^0.11.0",
```

Comment:
Should we use [gulp-gh-pages intead](https://www.npmjs.com/package/gulp-gh-pages)?